known-issues.yml

```diff
@@ -357,6 +357,15 @@ zib-Procedure-request:
       - cardinality: 0..1
         reason: It seems that the zib may have an error in the cardinality for this concept and the cardinality should acctually be 0..1 rather than 0..*. This issue is being tracked here https://bits.nictiz.nl/browse/ZIB-1488
 
+zib-Stoma:
+  zib deviations:
+    Condition.code:
+      - cardinality: 1..1
+        reason: The FHIR profiles are expected to implement zib cardinalities as conceptual cardinalities, but StomaType is essential to identify the Stoma. This aligns with the intention of the zib, as the StomaType concept is required for zib Stoma to make sense.
```

Comment:
As discussed in the team: it might be useful to ask the terminology team which code would be suitable for a "non categorized" stoma. At this point in time, with the knowledge we have, let's just stick with the solution you proposed.